### PR TITLE
Change ->. to #->.

### PR DESCRIPTION
**Pull request by facundominguez**

### CODE REVIEW

**Comment by facundominguez** [comment]
I'm testing this with the most recent build of the linear GHC: https://github.com/facundominguez/nixpkgs/compare/master...facundominguez:fd/ghc/linear-types

I could contribute a buildkite pipeline to build linear-base based on that fork of nixpkgs.

File: shell-stack.nix [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{ pkgs ?  import ./nixpkgs.nix {}
+, ghc ? pkgs.haskell.compiler.ghcLinearTypes20191220
+}:
+
+with pkgs;
+
+haskell.lib.buildStackProject {
+  name = "inline-java";
```
[the remark this answers is not in the document]
Comment (facundominguez) [reply shell-stack.nix]: oops

**Comment by flokli** [comment]
The clone URI still was `ssh`, and buildkite was stuck in confirming the fingerprint. It's changed now, feel free to push again :)

File: shell-stack.nix (hunk 1 above)
[the remark this answers is not in the document]
Comment (facundominguez) [reply shell-stack.nix]:
```suggestion
  name = "linear-base";
```

**Pull request merged**